ateam_ui/src/src/components/FieldComponent.vue

```diff
@@ -145,8 +145,8 @@
                 dragBoundFunc: dragBound,
                 r_id: robot.id,
                 team: robot.team,
-                x: robot.pose.position.x,
-                y: robot.pose.position.y,
+                x: robot.pose.position.y / 4,
```

Comment:
I don't think this is the best place to modify how the robots are positioned. What specifically are you trying to get the UI to do?

Reply:
If the robots are showing up in the wrong position on the UI then I think I need to set up the UI to also pull the field dimensions from the vision packet so that the scaling on the field and the robots match.

Reply:
It just had the wrong scaling. I'll revert this change for now and we can add that feature in later